buildroot-external/rootfs-overlay/usr/lib/systemd/system/raucdb-update.service

```diff
@@ -0,0 +1,12 @@
+[Unit]
+Description=Ensure rauc.db contains version information
+After=rauc.service
+Before=hassos-supervisor.service
+RequiresMountsFor=/mnt/data
```

Comment:
Can we maybe use

```
ConditionFirstBoot=yes
```

here? :thinking:  Ideally, I guess we need to check if it gets triggered after a device wipe too :thinking:

Reply:
I'd say you answered yourself, we'll also need to do that after the device wipe (or at least it's currently what I aimed for and makes more sense). And since we currently determine first boot condition from the presence of MACHINE_ID, it would IMHO only bring more complexity than how it's done now - i.e. ensure the version info is there on every boot.

I've reordered the script a bit more based on your feedback, so the overhead should be minimal now.

Reply:
We probably should clear the MACHINE_ID on device wipe :thinking: But this is a different discussion.